cmd/server-manager/typescript/src/RaceControl.ts

```diff
@@ -565,12 +565,26 @@ class LiveTimings implements WebsocketHandler {
 
         $(document).on("click", ".driver-link", this.toggleDriverSpeed.bind(this));
 
+        $(document).on("click", "#countdown", this.getFromButton.bind(this));
+
         $(document).on("submit", "#broadcast-chat-form", this.processChatForm.bind(this));
         $(document).on("submit", "#admin-command-form", this.processAdminCommandForm.bind(this));
         $(document).on("submit", "#kick-user-form", this.processKickUserForm.bind(this));
         $(document).on("submit", "#send-chat-form", this.processSendChatForm.bind(this));
     }
 
+    private getFromButton(e: ClickEvent): void {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const $target = $(e.currentTarget);
+        const $href = $target.attr("href")
```

Comment:
can you add semicolons to these lines please? also `$href` doesn't need a `$` prefix because it doesn't refer to a JQuery element, the value of `$href` is of type string